draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -392,6 +412,13 @@ const TitleBlockZen = ({
   navigationTabs,
   textDirection,
   surveyStatus,
+  titleAutomationId = "TitleBlock__Title",
+  avatarAutomationId = "TitleBlock__Avatar",
+  subtitleAutomationId = "TitleBlock__Subtitle",
+  sectionTitleAutomationId = "TitleBlock__SectionTitle",
+  sectionTitleDescriptionAutomationId = "TitleBlock__SectionTitleDescription",
+  breadcrumbAutomationId = "TitleBlock__Breadcrumb",
+  breadcrumbTextAutomationId = "TitleBlock__BreadcrumbText",
```

Comment:
Decided to always provide defaults based on what I was extending in a non-breaking way. I am unsure if this pattern follows what is used in Murmur but I feel that it is a safe bet.

Reply:
Great idea! LGTM